### PR TITLE
Update Create/Update methods to return the workflow

### PR DESCRIPTION
**Pull request by avivek**
Updated create and update org required workflow methods to return the required workflow object returned by the github api.
Merging this Fixes https://github.com/google/go-github/issues/2756

### CODE REVIEW

File: github/actions_required_workflows.go
Comment (gmlewis) [inline github/actions_required_workflows.go:101]: Let's please add a blank line before this one.

File: github/actions_required_workflows.go
Comment (gmlewis) [inline github/actions_required_workflows.go:106]: Let's please add a blank line before this one.

File: github/actions_required_workflows.go
Comment (gmlewis) [inline github/actions_required_workflows.go:138]: Let's please add a blank line before this one.

File: github/actions_required_workflows.go
Comment (gmlewis) [inline github/actions_required_workflows.go:143]: Let's please add a blank line before this one.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @avivek !
Just a few minor tweaks, please.
Then we will be ready for a second LGTM+Approval from any other contributor to this repo before merging.

File: github/actions_required_workflows.go
Comment (avivek) [reply github/actions_required_workflows.go:102]: Done.

File: github/actions_required_workflows.go
Comment (avivek) [reply github/actions_required_workflows.go:108]: Done.

File: github/actions_required_workflows.go
Comment (avivek) [reply github/actions_required_workflows.go:141]: Done

File: github/actions_required_workflows.go
Comment (avivek) [reply github/actions_required_workflows.go:147]: Done.

**Comment by avivek** [comment]
Thanks for your quick review @gmlewis .
I have made the changes, please take a look now.

**Review by gmlewis — APPROVED** [review]
Thank you, @avivek !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]

**Comment by gmlewis** [comment]
Thank you, @valbeat !
Merging.

**Pull request merged**
